distributed/cli/utils.py

```diff
@@ -1,30 +1,25 @@
-from tornado.ioloop import IOLoop
+import asyncio
+import logging
+import signal
 
+logger = logging.getLogger(__name__)
```

Comment:
We should probably either pass a logger into `wait_for_signal` or refactor the logging into the user of it.